Assets/MixedRealityToolkit/Providers/BaseInputDeviceManager.cs

```diff
@@ -12,6 +13,32 @@ namespace Microsoft.MixedReality.Toolkit.Input
     /// </summary>
     public abstract class BaseInputDeviceManager : BaseDataProvider<IMixedRealityInputSystem>, IMixedRealityInputDeviceManager
     {
+        private bool enablePointerCache = true;
+
+        /// <summary>
+        /// Control mechanism to enable/disable use of Pointer Cache in request/recycling of pointers by Input System
+        /// </summary>
+        public bool EnablePointerCache
+        {
+            get => enablePointerCache;
+            set
+            {
+                enablePointerCache = value;
+                if (!enablePointerCache)
```

Comment:
Do you want to ignore the change if enablePointerCache is already false?

Reply:
yea good idea, adding != check